app/javascript/bundles/MainApp/components/AppHeader.jsx

```diff
@@ -0,0 +1,54 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import TextFilter from './TextFilter';
+import SelectionFilter from './SelectionFilter';
+import AggregatedFilter from './AggregatedFilter';
+
+export default class AppHeader extends React.Component {
+  static minimalRatings = Array(3).fill(0).map((e, i) => i + 1);
+  static deliveryTimes = Array(8).fill(0).map((e, i) => 15 + i * 15);
+
+  constructor(props) {
+    super(props);
+  }
+
+  onFilterChanged = (filter, newFilterValue) => {
+    this.props.onFilterChange(filter, newFilterValue);
+  };
+
+  render() {
+    return <header>
+      <div className="row text-center">
+        <div className="col-12 text-center">
+          <h1>WeEat</h1>
+        </div>
+      </div>
+      <div className="row text-center" >
+        <div className="col-12 text-center">
+          <h4>We Want Eat All, We Want Eat Now!</h4>
+        </div>
+      </div>
+      <TextFilter filter={this.props.filters.filter[1]} onFilterChange={this.onFilterChanged} />
+      <div className="row justify-content-md-center selection-filters">
+        <SelectionFilter filter={this.props.filters.filter[2]}
```

Comment:
I think you could have used a object to map the filter values(items?) and title.
That could help "DRY up the code" and in the future enable us to add filters without copy pasting a bunch of places.

Reply:
I added a new function that generates the <SelectionFilter> tag, looks much shorter now
Thanks